CONTRIBUTING.md

```diff
@@ -0,0 +1,33 @@
+# GSF Member Contribution Guide
+
+Welcome to the GSF Carbon Aware SDK, and thank you for contributing to this project. This guide outlines the steps for working within the Carbon Aware SDK and the approved contribution process that members should follow.
+
+**This project is currently only looking for contributions from GSF Members. Please contact the project owners for individual contributions.**
```

Comment:
This first sentence is not true and can be deleted.  We're open to all contributors, but participation in the working group and meetings etc is only for GSF members.